docs/cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Should we keep this file? It's just a big comment?

Reply:
Yea, I thought we would use the auto generated support files in the future, but yea, it looks cleaner without them.

Removed!

Reply:
Note:

after removing these support files in https://github.com/0618/amplify-ui/commit/9f1fbf14c9acc054812d5f2c0c32d34c429b3342, build time reduced from 45+ min to 40- min.

|  Before | After  |
|---|---|
| ![image](https://user-images.githubusercontent.com/11983489/199128270-470ea7b8-5a07-4152-a335-de48dc1aba75.png)  | ![image](https://user-images.githubusercontent.com/11983489/199128251-824ed1fe-672c-44e0-9949-40749476ce88.png)  |